security/file_integrity_operator/file-integrity-operator-troubleshooting.adoc

```diff
@@ -54,14 +54,14 @@ Run:
 +
 [source,terminal]
 ----
-$ oc get pods -lapp=aide-ds-$(<FIO_NAME>)
+$ oc get pods -lapp=aide-ds-worker-fileintegrity
```

Comment:
update the label accordingly, should be:
oc -n openshift-file-integrity get pods  -lapp=worker-fileintegrity